src/css/footer.css

```diff
@@ -65,38 +120,56 @@
 
 .footer-list-menu .footer-item {
   position: relative;
-  transition: opacity 250ms ease;
 }
 
 .footer-list-menu .footer-item:after {
   position: absolute;
   content: '';
-  width: 37px;
+  width: 0px;
```

Comment:
why width: 0?

Reply:
for animation to showing on hover from left to right